bigtable/client/data.cc

```diff
@@ -19,6 +19,7 @@
 #include <thread>
 
 #include <absl/memory/memory.h>
+#include <bigtable/client/detail/bulk_mutator.h>
```

Comment:
What does the style guide say on `#include "..."` style vs. `#include <...>` and what are the differences? :-)

Should this project's headers be `#include "..."` and all other headers, whether C++ STL or other third-party headers be `#include <...>` or something else?

Reply:
I agree, this project headers should be `#include "..."`, and other projects should be `#include <...>`.

We have not been consistent so far, but never too late to start.  We should make a ruling about the generated `google/bigtable/data.pb.h` and siblings, are those part of this project? My inclination is no.